app/controllers/builds_controller.rb

```diff
@@ -54,12 +56,12 @@ def update
         if success
           redirect_to [current_project, @build]
         else
-          render :edit, status: 422
+          render :edit, status: :unprocessable_entity
         end
       end
 
       format.json do
-        render json: {}, status: success ? 200 : 422
+        render json: "{}", status: (success ? :ok : :unprocessable_entity)
```

Comment:
I'll try to revert ... I think something broke without it ... but yeah looks weird ...